src/app/calculator/compressed-air/air-leak/air-leak.component.ts

```diff
@@ -91,7 +92,7 @@ export class AirLeakComponent implements OnInit, AfterViewInit {
     this.airLeakService.calculate(this.settings);
     if (this.assessmentCalculator) {
       this.assessmentCalculator.airLeakInput = this.airLeakService.airLeakInput.getValue();;
-      this.saveAssessmentCalculator();
+      this.calculatorDbService.saveAssessmentCalculator(this.assessment, this.assessmentCalculator);
```

Comment:
For this to works properly `calculate()` will need to be marked `async` and `await` will have to be added to the `saveAssessmentCalculator()` call

Reply:
Investigate lag on changes to calculator when this is implemented.